app/survey_config/__init__.py

```diff
@@ -22,6 +23,7 @@
     "BusinessSurveyConfig",
     "NorthernIrelandBusinessSurveyConfig",
     "BEISBusinessSurveyConfig",
+    "NIBEISBusinessSurveyConfig",
```

Comment:
I know this is just copying the others, but now that there are more of these does it make sense to append `BusinessSurveyConfig` to all of these?  Might be clearer without it i.e just `NIBEIS`? Not sure what others think though. Also we now have a mismatch between `NI` and `NorthernIreland`

Reply:
With the NI bit do you think `NIBEISBusinessSurveyConfig` should be called `NorthernIrelandBEISBusinessSurveyConfig`? I just thought that was a bit long. Could change `NorthernIrelandBusinessSurveyConfig` to `NIBusinessSurveyConfig` though?

As for the `BusinessSurveyConfig` bit Im happy to reduce it to `SurveyConfig` or even `config`? I think removing it completely would make it a bit less clear what was being used here

Reply:
Personally I'm fine with NI for these

Reply:
I think you should flip them so it is `BEISNI` to match `CensusNisra`

Reply:
Done the NI stuff 👍